src/v/cluster/partition_manager.cc

```diff
@@ -115,26 +115,52 @@ ss::future<consensus_ptr> partition_manager::manage(
   raft::keep_snapshotted_log keep_snapshotted_log) {
     gate_guard guard(_gate);
     auto dl_result = co_await maybe_download_log(ntp_cfg, rtp);
-    auto [logs_recovered, min_kafka_offset, max_kafka_offset, manifest]
+    auto
+      [logs_recovered,
+       clean_download,
+       min_offset,
+       max_offset,
+       manifest,
+       ot_state]
       = dl_result;
     if (logs_recovered) {
         vlog(
           clusterlog.info,
           "Log download complete, ntp: {}, rev: {}, "
-          "min_kafka_offset: {}, max_kafka_offset: {}",
+          "min_offset: {}, max_offset: {}",
           ntp_cfg.ntp(),
           ntp_cfg.get_revision(),
-          min_kafka_offset,
-          max_kafka_offset);
+          min_offset,
+          max_offset);
+        if (!clean_download) {
+            vlog(
+              clusterlog.warn,
```

Comment:
nit: can this be raised to an ERROR? This seems like an actionable message that could be monitored for

Reply:
it can but I want to avoid that at least for now, to keep the change local
otherwise it will be difficult to backport